apps/educational_institutions/models.py

```diff
@@ -63,28 +63,34 @@ def __repr__(self):
                f'faculty_id="{self.faculty_id}")'
 
 
-class Dekan(Base):
-    __tablename__ = 'dekan'
+class Dean(Base):
+    __tablename__ = 'dean'
 
-    dekan_id = Column(INTEGER, primary_key=True, nullable=False)
-    full_name = Column(VARCHAR(length=255), nullable=False)
+    dean_id = Column(INTEGER, primary_key=True, nullable=False)
+    last_name = Column(VARCHAR(length=50), nullable=False)
+    first_name = Column(VARCHAR(length=50), nullable=False)
+    middle_name = Column(VARCHAR(length=50))
 
-    faculty = relationship("Faculty", back_populates="dekan")
+    faculty = relationship("Faculty", back_populates="dean")
 
     def __repr__(self):
-        return f'{self.__class__.__name__}(dekan_id="{self.dekan_id}", full_name="{self.full_name}")'
+        return f'{self.__class__.__name__}(dean_id="{self.dean_id}", last_name="{self.last_name}",' \
+               f'first_name="{self.first_name}", middle_name="{self.middle_name}")'
 
 
 class Rector(Base):
     __tablename__ = 'rector'
 
     rector_id = Column(INTEGER, primary_key=True, nullable=False)
-    full_name = Column(VARCHAR(length=255), nullable=False)
+    last_name = Column(VARCHAR(length=50), nullable=False)
+    first_name = Column(VARCHAR(length=50), nullable=False)
+    middle_name = Column(VARCHAR(length=50))
 
     university = relationship("University", back_populates="rector")
 
     def __str__(self):
-        return f'{self.__class__.__name__}(rector_id="{self.rector_id}",full_name="{self.full_name}")'
+        return f'{self.__class__.__name__}(rector_id="{self.rector_id}", last_name="{self.last_name}",' \
```

Comment:
Should we use: First -> Middle -> Last names
E.G. Ivan Ivanovich Ivanchenko

Reply:
fixed